src/components/journeys/JourneyInstancesDataTable/index.tsx

```diff
@@ -66,17 +65,8 @@ const JourneyInstancesDataTable: FunctionComponent<JourneysDataTableProps> = ({
         onColumnResize={(params) => {
           setColumnWidth(params.colDef.field, params.width);
         }}
-        onFilterModelChange={(model) => {
-          if (!isEqual(model, modelGridProps.filterModel)) {
-            modelGridProps.onFilterModelChange(model);
-          }
-        }}
-        onSortModelChange={(model) => {
-          // Something strange going on here with infinite state updates, so I added the line below
-          if (!isEqual(model, modelGridProps.sortModel)) {
-            modelGridProps.onSortModelChange(model);
-          }
-        }}
+        onFilterModelChange={modelGridProps.onFilterModelChange}
+        onSortModelChange={modelGridProps.onSortModelChange}
```

Comment:
Do you feel that this is better than passing these (and the rest of the `modelGridProps`) as `{...modelGridProps}`?

I'm not sure what I think. This is clearer, but also more verbose.

Reply:
I prefer clearer and more verbose, but I also see that this way of doing it is a break from how it's done in other components, so now that I think about it again I'm with you and change it!